### ISSUE TITLE
[Suggestion] Allows villagers to put modded crops/seeds into their composter

### ISSUE THREAD

**Issue by lzm11111111** [MehVahdJukaar/SmarterFarmers#18]
If the extra seeds in the villagers' inventory are not used, their inventory will be full sooner or later. In the vanilla, they only put wheat and beetroot seeds into the composter, they can't put modded seeds into the composter, I hope you can fix this issue. Thanks.

**Comment by MehVahdJukaar** [comment]
this is a forge feature actually. I assume you are on fabric. in that case that was already on my todo list

**Comment by lzm11111111** [comment]
Yes i'm using fabric and I also found another bug that the farmers behavior seems doesn't functioning on neoforge version.
